### PR TITLE
Bug 1732150: Update Scheduler config description

### PR DESCRIPTION
**Pull request by damemi**
This adds a more unique description to the scheduler config, bz https://bugzilla.redhat.com/show_bug.cgi?id=1732150

### CODE REVIEW

File: config/v1/types_scheduling.go [hunk 1]
```diff
@@ -6,7 +6,11 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// Scheduler holds cluster-wide information about Scheduler.  The canonical name is `cluster`
+// Scheduler holds cluster-wide config information about the Kubernetes Scheduler.
+// It enables specification of a policy configmap, a default node label selector,
```
Comment (ravisantoshgudimetla) [inline config/v1/types_scheduling.go]: Do we need to add more info on the fields here? It becomes difficult whenever we add new fields right? Why can't we add the documentation at the individual fields and should be more specific in this section?

Comment (damemi) [reply config/v1/types_scheduling.go]: @ravisantoshgudimetla this is the description for the top-level config object giving a general overview of what it holds. The individual fields do have documentation but they aren't visible for example when doing `oc explain` on this type, or in the Console (see screenshot in the BZ)

Comment (damemi) [reply config/v1/types_scheduling.go]: We don't need the info on the fields here, but I'm open to suggestions for how to improve this description beyond `Scheduler holds cluster-wide config information about the Kubernetes scheduler`

Comment (ravisantoshgudimetla) [reply config/v1/types_scheduling.go]: I saw the screenshot in the BZ, I believe something on the lines of

`Scheduler holds cluster-wide config information to run the Kubernetes Scheduler and influence its placement decisions. The canonical name for this config is cluster`

And perhaps we need to have drill down further in the UI which talks about specific fields

Comment (damemi) [reply config/v1/types_scheduling.go]: I think that sounds great, updated. thanks Ravi

**Pull request merged**